sdk/playwrighttesting/microsoft-playwright-testing/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# 1.0.0-beta.1 (2024-07-31)
+
+## Added
```

Comment:
The changelog here needs to match the formatting in other packages. I know it can feel arbitrary but we have a bunch of automation that relies on a certain structure. Can you look at other packages in the repo and match the headers and their nesting level?